index.html

```diff
@@ -83,54 +87,98 @@
     <div id="chart"></div>
   </div>
 </div>
-<script type="text/javascript" src="https://npmcdn.com/d3@3.5.17/d3.min.js"></script>
+    
+    <script type="text/javascript" src="https://npmcdn.com/d3@3.5.17/d3.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/science@1.9.3/science.v1.min.js"></script>
 <script type="text/javascript" src="https://npmcdn.com/simple-statistics@2.0.0/dist/simple-statistics.min.js"></script>
 <script>
   var data = [];
   var options = {
     yearRange: 'all'
   };
-
-  var svg;
-
+  var svg, x, y;
+  var margin = { top: 15, right: 15, bottom: 100, left: 70 };
+  var width = 800 - margin.right - margin.left;
+  var height = 600 - margin.top - margin.bottom;
+    
   // FETCH DATA
-
+  d3.json('data-cpi2.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
   function setup() {
     // SVG
-
     svg = d3.select('#chart')
-      .append('svg');
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.FEDFUNDS; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.CPI; })])    
+      .range([height, 0])    
+      .nice();
+      
     // AXES
-
+      var xAxis = d3.svg.axis().scale(x).orient("bottom");      
+      var yAxis = d3.svg.axis().scale(y).orient("left") ;   
+      
+      svg.append('g')
+          .attr('class', 'x axis')
+          .call(xAxis)
+          .attr('transform', 'translate(0,' + (height) + ')')
+        .append('text')
+          .attr('x', width/2)
+          .attr('y', 70)
+          .style('text-anchor', 'middle')
+          .text('Federal Funds Interest Rate, percent');      
+      
+      svg.append('g')
+          .attr('class', 'y axis')
+          .call(yAxis)
+        .append('text')
+          .attr('transform', 'rotate(-90)')
+          .attr('y', -60)
+          .attr('x', -220)      
```

Comment:
We talked about this before class, but we should be able to set `x` to `-height / 2`. The `y` position we'll have to set manually based on how wide our tick labels are.